README.md

```diff
@@ -108,8 +108,6 @@ Otherwise, report the issue to the package maintainer!
 * [Homebrew](https://formulae.brew.sh/cask/logisim-evolution) (`brew install --cask logisim-evolution`)
 * [MacPorts](https://ports.macports.org/port/logisim-evolution/details/) (`port install logisim-evolution`)
 * [Chocolatey](https://community.chocolatey.org/packages/logisim-evolution) (`choco install logisim-evolution`)
-* [winget](https://wingetgui.com/apps?id=Logisim-evolution.Logisim-evolution)
-(`winget install --id=Logisim-evolution.Logisim-evolution  -e`)
```

Comment:
Indeed, the above URL is dead, an Logisim-evolution cannot be anymore found on that website, but [its manifest](https://github.com/microsoft/winget-pkgs/tree/master/manifests/l/logisim-evolution/logisim-evolution) is still in the winget package repository and it is also listed on [repology.org](https://repology.org/project/logisim-evolution/versions). Instead of removing this entry, I would suggest updating the URL.

Reply:
I updated the URL.
GitHub did not realize this change.
Could you please mark this as done so that this PR can get merged?